test/ng/directive/ngModelSpec.js

```diff
@@ -1784,6 +1807,63 @@ describe('ngModelOptions attributes', function() {
   }));
 
 
+  it('should inherit options from ngModelOptions directives declared on ancestor elements', function() {
+    var container = $compile('<div ng-model-options="{ allowInvalid: true }">' +
+               '<form ng-model-options="{ updateOn: \'blur\' }">' +
+                 '<input ng-model-options="{ updateOn: \'default\' }">' +
```

Comment:
Did you want to check the value of this one, too or are they there for a particular reason?